lib/src/commands/check_duplicate.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:arb_utils/arb_utils.dart';
+import 'package:arb_utils/src/primitives/check_duplicate.dart';
+import 'package:args/command_runner.dart';
+import 'package:dcli/dcli.dart';
+
+class CheckDuplicateCommand extends Command {
+  @override
+  String get name => 'duplicate';
+
+  @override
+  String get description => 'Check for duplicate values in the arb file.';
+
+  @override
+  String get invocation =>
+      '${super.invocation} <left-arb-file> <...arb-files-to-merge>';
+
+  CheckDuplicateCommand();
+
+  @override
+  FutureOr<void> run() async {
+    if (argResults!.rest.isEmpty) {
```

Comment:
add a check on argResults not being null and ~throwing an UnsupportedError~ printting the error and exiting with 1 if the check fails rather than using an unsafe ! opertor directly

Reply:
extract variable...
`final filePaths = argResults.rest;`

Reply:
check for `filePaths` not being empty or ~throw an ArgumentError~ print the pertinent error and exit with 2